app/src/main/java/com/cobresun/brun/pantsorshorts/presenter/MainActivityPresenter.java

```diff
@@ -70,32 +81,60 @@ public MainActivityPresenter(MainActivityView view, UserDataRepository userDataR
         this.mContext = context;
     }
 
-    private void updateUserThreshold(int howTheyFelt, float currentTemp) {
+    private void updateUserThreshold(int howTheyFelt) {
         if (howTheyFelt == COLD) {
-            userDataRepository.writeUserThreshold(currentTemp + 1);
+            int curPreference = userDataRepository.readUserThreshold();
+            while (pantsOrShorts(curPreference) == SHORTS){
+                curPreference++;
```

Comment:
Brand new logic?

Reply:
Yea wait what is this even doing

Reply:
Logic to ensure that the suggestion changes when the user clicks the button. If you would like to include some of the old logic, we can start the new threshold at the current temperature and go up/down from there.